clang/bindings/python/clang/cindex.py

```diff
@@ -3263,9 +3298,11 @@ def reparse(self, unsaved_files=None, options=0):
             unsaved_files = []
 
         unsaved_files_array = self.process_unsaved_files(unsaved_files)
-        ptr = conf.lib.clang_reparseTranslationUnit(
+        result = conf.lib.clang_reparseTranslationUnit(
             self, len(unsaved_files), unsaved_files_array, options
         )
+        if result != 0:
+            raise CXError(result, 'Error reparsing TranslationUnit.')
```

Comment:
Since we are now explicitly throwing an error where there was none before, this should probably haven an entry in `clang/docs/ReleaseNotes.rst` under "Potentially Breaking Changes"